custom/ewsghana/api.py

```diff
@@ -273,6 +285,18 @@ def location_sync(self, ews_location):
 
             if apply_updates(location, location_dict):
                 location.save()
+            for loc in ews_location.supply_points:
```

Comment:
In EWS, we have this situation, NATIONAL have two supply point's 'Komfo Anokye Teaching Hospital' and 'Korle Bu Teaching Hospital', but in HQ we creating the new locations for these.